### DISCUSSION TITLE
Modular configuration of kafka listeners

### DISCUSSION THREAD

**Discussion by ErikEngerd** [strimzi/strimzi-kafka-operator#10883] [Q&A]
In our situation, we have a kafka installation based on strimzi to which we want to add new listeners. 

Organisationally, the kafka resource is delivered by another organization. We can add a listener to that resource but that breaks the organization since then who is the owner of that resource? 

So, is it possible to add a new listener to a kafka cluster by adding a new resource instead of modifying an existing one? In principle I would only like to add a new listener for an existing cluster and keep the existing kafka resource unmodified. 

Is this possible?

**Comment by scholzj** [comment]
No, the listener configuration is part of the Kafka CR.
